include/openPMD/IO/Access.hpp

```diff
@@ -20,18 +20,65 @@
  */
 #pragma once
 
+#include <stdexcept>
+
 namespace openPMD
 {
 /** File access mode to use during IO.
  */
 enum class Access
 {
     READ_ONLY, //!< open series as read-only, fails if series is not found
+    READ_RANDOM_ACCESS = READ_ONLY,
+    READ_LINEAR,
```

Comment:
Maybe deprecate:
```cpp
enum class Access
{
    READ_RANDOM_ACCESS, //!< open series as read-only, fails
                        //!< if series is not found
    READ_ONLY [[deprecated("Renamed as READ_RANDOM_ACCESS")]] =
        READ_RANDOM_ACCESS,
    READ_LINEAR,
    READ_WRITE, //!< open existing series as writable
    CREATE, //!< create new series and truncate existing (files)
    APPEND //!< write new iterations to an existing series without reading
}; // Access
```

Would then need to change every occurrence in the code base.